quadkey/bq/doc/REFERENCE.md

```diff
@@ -147,4 +147,19 @@ quadkey.ST_GEOGFROMQUADINT_BOUNDARY(quadint INT64) -> GEOGRAPHY
 
 * `quadint`: `INT64` quadint we want to get the boundary geography from.
 
-Returns the geography boundary for a given quadint. We extract the boundary the same way as when we calculate the bbox, then enclose it in a GEOJSON and finally transform it to geography.
\ No newline at end of file
+Returns the geography boundary for a given quadint. We extract the boundary the same way as when we calculate the bbox, then enclose it in a GEOJSON and finally transform it to geography.
+
+### quadkey.LONGLAT_ASQUADINTLIST
```

Comment:
I would add `_RESOLUTION` or something like that here, since you are calculating a subcell of the tile (based on the resolution).